linkerd/app/inbound/src/http/router.rs

```diff
@@ -60,15 +60,6 @@ struct ClientRescue;
 struct LogicalError {
     logical: Option<NameAddr>,
     addr: Remote<ServerAddr>,
-    http: http::Version,
-    #[source]
-    source: Error,
-}
-
-#[derive(Debug, thiserror::Error)]
-#[error("route ({:?}): {}", .route_labels, .source)]
-struct RouteError {
```

Comment:
I only added this error type because the ticket for error contexts listed "logical/route/concrete/endpoint" as the desired contexts to include in errors...do we not want to add a route error context after all, or is that still something we care about?

Reply:
Gotcha. There probably wasn't much thought behind that description.

> Remove RouteError types for now. They're not really critical for debugging. We may restore them later as those stacks evolve.

I think the general service/endpoint coordinates are more sufficient for what we really need for diagnostics/error reporting. I want to hold off on doing route-specific things until we see how those types start to settle. I suspect we may want to omit them in the short term--they'll add a ton of verbosity to already-busy error messages. This could change as routes need to actually fail requests with errors, which I don't believe is really possible (other than load shedding on service availability).

Reply:
cool, sounds good to me!